.vscode/launch.json

```diff
@@ -26,6 +26,15 @@
       ],
       "console": "integratedTerminal",
     },
+    {
```

Comment:
This should be in .vscode/tasks.json given it's not a long-running command

Reply:
No, it's not a task, it's a debug configuration.

But I'm happy to drop this commit if you don't like it, just thought it would be convenient next time. I can also keep it in a stash or something.

Reply:
Oh I see. In that case let's keep it